chart/templates/flower/flower-deployment.yaml

```
@@ -111,6 +111,10 @@ spec:
                   name: {{ template "flower_secret" . }}
                   key: basicAuth
             {{- end }}
+            {{- if .Values.flower.url_prefix }}
+            - name: FLOWER_URL_PREFIX
+              value: {{.Values.flower.url_prefix }}
+            {{- end }}
```

Ah.

Instead of adding a new setting for this, we should
```suggestion
            {{- include "custom_airflow_environment" . | indent 10 }}
```

Then the existing environment variable would work. Right now without any changes this can be set in the airflow.cfg config map.

---

Ok, I've dropped all other changes and just added this include.